lxd/device/config/devices.go

```diff
@@ -88,6 +88,10 @@ func NewDevices(nativeSet map[string]map[string]string) Devices {
 	for devName, devConfig := range nativeSet {
 		newDev := Device{}
 		for k, v := range devConfig {
+			if v == "" {
```

Comment:
I'm not sure why do we need this change - it feels like it may be masking a bug/issue elsewhere?

Reply:
From what I could see the only case where this is useful is when creating a device from a block volume because the path is empty. Even so I think it makes sense to remove it because it can unintentionally mask a bug in the future.